assets/javascripts/discourse/components/share-report.hbs

```diff
@@ -0,0 +1,19 @@
+<div role="button" onclick={{action "open"}}>
```

Comment:
Could we make this a real `<button>` or `<DButton>`? `div role=button feels like a bit of a hack.

Either way, let's switch to the `{{on}}` modifier for the click event

```suggestion
<div role="button" {{on "click" this.open}}>
```

Reply:
Sorry for the confusion. As you pointed out there was the old hbs file left in place that I will remove. We need to keep this as is for styling purposes as it is expected to be only displayed as text. I could do something like 

```
  <a href="#" onClick={{this.open}} class="share-report-button">
```

that feels a little more clean.

Reply:
I see, ok. Happy with whichever you prefer. If we go with the `<a` approach, then we'll have to add `event.preventDefault()` to the handler